modules/openapi-generator/src/main/resources/csharp-netcore/IReadableConfiguration.mustache

```diff
@@ -89,5 +91,15 @@ namespace {{packageName}}.Client
         /// <param name="apiKeyIdentifier">API key identifier (authentication scheme).</param>
         /// <returns>API key with prefix.</returns>
         string GetApiKeyWithPrefix(string apiKeyIdentifier);
+		
+		/// <summary>
+		/// Gets the json serializer settings
+		/// </summary>
+		JsonSerializerSettings SerializerSettings { get; }
+		
+		/// <summary>
+		/// Gets the proxy
+		/// </summary>
+		WebProxy Proxy { get; }
```

Comment:
Note: changing this interface is a breaking change for existing client consumers.
Also, formatting is off by 4 spaces too many here.

Reply:
I thought this interface is just used from the `Configuration` ? And this class is also generated. Using this proxy is optional

Reply:
Since 5.0 is released soon, I think the breaking change here will be fine for client release. I still think the JsonSerializerSettings should stay in ApiClient, however, because this isn't API configuration but an implementation detail of ApiClient. Since we allow for users to define their own ApiClients (and bring their own serializers), it doesn't make a lot of sense to put ApiClient implementation-specific details into the Configuration object. That is, this configuration object should refer to `System` namespaces only.

I'd be happy to resolve this in a separate commit.